### PR TITLE
Add BleachField to contact form and model

### PR DESCRIPTION
**Pull request by amakarudze**
Addresses #767.

### CODE REVIEW

File: djangogirls/settings.py [hunk 1]
```diff
@@ -274,3 +275,15 @@ def gettext(s):
 RECAPTCHA_TESTING = os.environ.get("RECAPTCHA_TESTING") == 'True'
 if RECAPTCHA_TESTING:
     SILENCED_SYSTEM_CHECKS = ['captcha.recaptcha_test_key_error']
+
+BLEACH_ALLOWED_TAGS = ['p', 'b', 'i', 'u']
+
+BLEACH_ALLOWED_ATTRIBUTES = ['title']
+
+BLEACH_ALLOWED_PROTOCOLS = [
```
Comment (marksweb) [inline djangogirls/settings.py:283]: I don't think this setting is needed because links aren't allowed.

**Review by marksweb — CHANGES_REQUESTED** [review]
I've added a few comments.

File: djangogirls/settings.py (hunk 1 above)
Comment (amakarudze) [reply djangogirls/settings.py:283]: Ok, will remove this as well.

**Comment by ramonsaraiva** [comment]
Do we still want to bleach the message field? I believe it is only missing adding that unit test check, right?

**Comment by marksweb** [comment]
@ramonsaraiva yeah I think it's still a worthwhile addition.

**Comment by ramonsaraiva** [comment]
@marksweb I added changes to the comments you've made here https://github.com/DjangoGirls/djangogirls/pull/834 - closing this draft.

**Pull request closed without merging**